packages/website/pages/components/toast.mdx

```diff
@@ -178,6 +178,22 @@ const App = () => (
 );
 ```
 
+### Disable the countdown bar
+
+Use the `showCountdownBar` prop to toggle the visibility of the countdown bar.
+
+```.jsx
```

Comment:
Maybe we should also add usage to the theme config of toast to demonstrate disabling the countdown bar globally.

```js
import { ThemeProvider } from 'fannypack';

const theme = {
  Toast: {
    defaultProps: {
      showCountdownBar: false
    }
  }
}

const App = () => (
  <ThemeProvider theme={theme}>
    // ... your app
  </ThemeProvider>
);
```

Reply:
Yeah that is a good idea! Just updated this.